src/Plugin/GraphQL/Schema/SdlSchemaPluginBase.php

```diff
@@ -111,6 +118,13 @@ public function __construct(
     $this->moduleHandler = $moduleHandler;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function setServerId(string $serverId): void {
+    $this->serverId = $serverId;
+  }
+
```

Comment:
Instead of using a setter to configure the server ID, I'm wondering if it wouldn't be better to make the plugin configurable by implementing <code>ConfigurableInterface</code>. In fact, both schema plugins we ship _are_ already configurable (but the base class `SdlSchemaPluginBase` isn't, and changing this would be a B/C break).